extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -889,7 +889,8 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 					dataSource: currentConnection?.serverName!,
 					authentication: this._authenticationType,
 					userName: this._sqlServerUsername,
-					password: this._sqlServerPassword
+					password: this._sqlServerPassword,
+					trustServerCertificate: currentConnection?.options.trustServerCertificate ?? false
```

Comment:
we should ensure we consider this in the other source connections we are making.  I added this parameter for validation as well:
const trustServerCertificate = currentConnection?.options['trustServerCertificate'] === true;